Notes on renamed pickers (#6317)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/document-picker.md

`Returns: IPublishedContent`

The Document Picker opens a panel to pick a specific page from the content structure. The value saved is the selected nodes [UDI](../../../../reference/querying/udi-identifiers.md).

{% hint style="info" %}

> The Document Picker is was what known as the **Content Picker** in version 13 and below.

Reviewer: ```suggestion
The Document Picker was formerly known as the **Content Picker** in version 13 and below.
```
